.github/workflows/testing.yml

```diff
@@ -59,6 +59,9 @@ jobs:
       - name: Install sycamore
         run: poetry install --all-extras
         working-directory: lib/sycamore
+      - name: Download nltk packages
+        run: poetry run python -m nltk.downloader punkt_tab averaged_perceptron_tagger_eng
+        working-directory: lib/sycamore
```

Comment:
why do we need to do this twice?

Reply:
Once for unit test, once for integration. Was breaking otherwise.